pennylane/ops/qubit/matrix_ops.py

```diff
@@ -374,8 +374,13 @@ def pow(self, z):
 
     def _controlled(self, wire):
         ctrl_wires = self.control_wires + wire
+        old_control_values = self.hyperparameters["control_values"]
+        values = None if old_control_values is None else f"{old_control_values}1"
```

Comment:
Why is 1 added after `old_control_values`? Also is `self.hyperparameters["control_values"]` deprecated?

Reply:
`"1"` is the control value for the new control wire. Since the new `wire` is appended to the end of the already existing control wires, the new control value is also appended to the end.

As far as I can tell right now, `self.hyperparameters['control_values']` is the one way to retrieve the control values from `ControlledQubitUnitary`.

Reply:
Ok great thank you for the details !